framework/app.mk

```diff
@@ -5,6 +5,8 @@
 # list of application-wide excluded source files
 excluded_srcfiles :=
 
+print-%  : ; @echo $* = $($*)
```

Comment:
Just a target used for debugging, I will remove it.
It allows you to do something like `make print-depend_dirs` and it prints out the variable `depend_dirs`.